### ISSUE TITLE
Cannot get websocket work with certain endpoint 

### ISSUE THREAD

**Issue by HarleyAppleChoi** [forbole/bdjuno#55]
## Bug description
When the config.toml file input with the corresponding rpc and lcd node, the websocket didn't work.

## Steps to reproduce
In .toml file change the rpc and client node to the following:
`rpc_node = "https://rpc.stargate.bigdipper.live:443"`
`client_node = "https://api.stargate.bigdipper.live:443"`

It gives out the error message :
`ERROR: websocket: bad handshake: failed to start RPC client [desmos-labs/juno@v0.0.0-20200910085615-790d94b23c6e/parse/handler.go:38]`

## Expected behavior
The rpc node should work as well as on desmos.

**Comment by RiccardoM** [comment]
This looks like something is wrong with the configuration of the RPC node. I've tried using port `80` and `443`, as well as protocol `http` and `https`. Are we sure the proper ports are open there? Maybe @kwunyeung you can take a look at the server configuration or tell us how you connect there with BigDipper?

**Comment by kwunyeung** [comment]
Yes ports are opened and headers are passed through. I have tested websocket can be connected with `wss://rpc.stargate.bigdipper.live/websocket`.

This is the websockect echo tool I use

https://www.websocket.org/echo.html

**Comment by RiccardoM** [comment]
Since this is more related to Juno, I've moved the discussion inside desmos-labs/juno#6.

*TL;DR*. Unfortunately there is nothing much we can do until a stable version of Cosmos v0.40 is out and a documentation on how to migrate to it is available.
